airbyte-commons-protocol/src/main/java/io/airbyte/commons/protocol/AirbyteMessageMigrator.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.protocol;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.airbyte.commons.protocol.migrations.AirbyteMessageMigration;
+import io.airbyte.commons.version.AirbyteVersion;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.SortedMap;
+import java.util.TreeMap;
+import javax.annotation.PostConstruct;
+import javax.inject.Singleton;
```

Comment:
FYI, I created the PR to switch to the Jakarta annotations package:  https://github.com/airbytehq/airbyte/pull/17251